blackbox/docs/appendices/release-notes/4.0.0.rst

```diff
@@ -0,0 +1,424 @@
+.. _version_4.0.0:
+
+=============
+Version 4.0.0
+=============
+
+Released on 2019/06/25.
+
+.. NOTE::
+
+    If you are upgrading a cluster, you must be running CrateDB 3.0.4 or higher
+    before you upgrade to 4.0.0.
+
+    We recommend that you upgrade to the latest 3.3 release before moving to
+    4.0.0.
+
+    Any upgrade from a version prior to :ref:`version_4.0.0` will require
```

Comment:
```suggestion
    An upgrade to :ref:`version_4.0.0` requires a full restart upgrade
```

`prior to 4.0.0` is a given since users cannot be on `4.0.0` yet and upgrade to `4.0.0`